src/main/resources/templates/application-integrationtest.properties

```diff
@@ -0,0 +1,2 @@
+spring.datasource.url = jdbc:h2:mem:test
```

Comment:
This shouldn't be here

Reply:
Please review again, this branch wasn't update with main.